### ISSUE TITLE
Medical Nanobots Don't Fix Broken Bones

### ISSUE THREAD

**Issue by falcon2346** [d3athrow/vgstation13#36973]
#### Description of issue
Medical Nanobots don't cure broken bones even though I'm fairly certain they're supposed to. They also fix Internal Bleeds but much like Bicaridine overdoses they don't remove it from the scanner. However, unlike Bicaridine od's, the IB can still be fixed with surgery, whereas with Bicaridine od's the IB simply disappears after the retractor step.

#### Difference between expected and actual behavior
Expected: Take nanobots and expect my broken bones to feel better after a few moments.
Actual: my bones remain broken until operated on.

#### Steps to reproduce
Break a bone, take medical nanobots

#### Length of time in which bug has been known to occur
No idea, medical nanobots don't actually get used very often even when they're made so it may have been an issue for quite some time.

#### Server revision
Game: 277ca7e95774514528f86765c50a7fdfb4d9159a 
Patches: e133d83e20999047f0766391eac1fcb5954219f8 
Current Infomational Settings: 
BYOND version of server: 515.1620
Protect Authority Roles From Traitor: 1

**Comment by SECBATON-GRIFFON** [comment]
eh, i made hivelord hearts for that im pretty sure

**Comment by hacker-on-steroids** [comment]
https://github.com/vgstation-coders/vgstation13/blob/2d96985b00e91c22114d98997e7bff6058a1f387/code/modules/reagents/reagents/reagents_medical.dm#L1041-L1047
i reckon these 3 flags don't do anything for internal organs

**Comment by SECBATON-GRIFFON** [comment]
> i reckon these 3 flags don't do anything for internal organs

butchered the lazytown quote award

**Comment by hacker-on-steroids** [comment]
no lazytown in romania

**Comment by wotisjanitor** [comment]
I think this should be changed honestly.

**Comment by Killette2** [comment]
Bones have always been a bitch to fix. Hivelord, surgery, and potions of rejuv are the only surefire ways to fix bones without fuckery on the level of staff of change. I am currently unsure if changeling rejuv, vampire rejuv, or vampire coffins each respectively fix these

Due to the task of obtaining medical nanobots I almost want to justify this but it's very on the fence to have a self bone healing treatment better than splints. There has never been such a reliable method before since hivelord hearts expire

**Comment by falcon2346** [comment]
I'd hardly even call it "self bone healing" since you need like 3 departments to cooperate to even make it. Yes once you've made the nanobots you could take a pill to fix your bones, but there's an entire process to getting to that point, making medical nanobots on your own isn't exactly easy. It also seems to me like they were intended to fix broken bones since as @hacker-on-steroids said those three flags there aren't doing anything for organs.

**Comment by Killette2** [comment]
Science can make the metals with both xenobio since they're already making black slimes and their xenobotany lab reliably without ever touching a pickaxe while using a chem dispenser. Docs delight also from xenobio or their botany lab if needed. If you were meaning mining and medical/kitchen/botany would be needed science has everything they have but better at the start of the round. Stretching that a player has to do the job of botany isn't much with how easy it's been made where you can set and forget your potency grass while you're in the middle of feeding slimes

tl;dr, sometimes you have to tell science no when it comes to getting a new toy
